### PR TITLE
Changelog: update with recent PRs.

### PR DESCRIPTION
**Pull request by patrickvacek**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,31 @@
...
+- Do not provision if the Primary times out while connecting to Secondaries: [PR](https://github.com/advancedtelematic/aktualizr/pull/1491)
+- Check and fetch root metadata according to the Uptane standard: [PR](https://github.com/advancedtelematic/aktualizr/pull/1501)
+- Don't fetch Snapshot or Targets metadata if we already have the latest: [PR](https://github.com/advancedtelematic/aktualizr/pull/1503)
+- Dynamically link aktualizr and the tests with libaktualizr as shared library: [PR](https://github.com/advancedtelematic/aktualizr/pull/1512)
+- Reject all targets if one doesn't match: [PR](https://github.com/advancedtelematic/aktualizr/pull/1510)
+- Updated code contribution guidelines: [PR](https://github.com/advancedtelematic/aktualizr/pull/1507)
+
+### Fixed
+
+- Do not provision if the Primary times out while connecting to Secondaries: [PR](https://github.com/advancedtelematic/aktualizr/pull/1491)
```
Comment (lbonn) [inline CHANGELOG.md]: It's also in `Changed`. Maybe just keep it in `Fixed`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,10 +2,31 @@
...
 ## [??? (unreleased)]
 
+
+## [2020.1] - 2020-01-16
+
+### Changed
+
+- Increased Targets metadata file size limit: [PR](https://github.com/advancedtelematic/aktualizr/pull/1476)
+- Do not provision if the Primary times out while connecting to Secondaries: [PR](https://github.com/advancedtelematic/aktualizr/pull/1491)
+- Check and fetch root metadata according to the Uptane standard: [PR](https://github.com/advancedtelematic/aktualizr/pull/1501)
```
Comment (lbonn) [inline CHANGELOG.md]: Aren't these more fixes (1501, 1510)?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,10 +2,31 @@
...
+- Don't fetch Snapshot or Targets metadata if we already have the latest: [PR](https://github.com/advancedtelematic/aktualizr/pull/1503)
+- Dynamically link aktualizr and the tests with libaktualizr as shared library: [PR](https://github.com/advancedtelematic/aktualizr/pull/1512)
+- Reject all targets if one doesn't match: [PR](https://github.com/advancedtelematic/aktualizr/pull/1510)
+- Updated code contribution guidelines: [PR](https://github.com/advancedtelematic/aktualizr/pull/1507)
```
Comment (lbonn) [inline CHANGELOG.md]: Maybe not changelog worthy (library users should not care)

File: CHANGELOG.md (hunk 1 above)
Comment (patrickvacek) [reply CHANGELOG.md]: Oops, didn't mean to duplicate it. Fixed.

File: CHANGELOG.md (hunk 3 above)
Comment (patrickvacek) [reply CHANGELOG.md]: Fair point. Removed.

File: CHANGELOG.md (hunk 2 above)
Comment (patrickvacek) [reply CHANGELOG.md]: I debated that. I argue that nothing was broken before in either case. I think both items are definite improvements, but to me, "fix" implies that there was a bug.

Comment (lbonn) [reply CHANGELOG.md]: Ok, fair enough.

**Comment by patrickvacek** [comment]
Rebased and ready for review.

**Pull request merged**